tests/robot-tests/tests/libs/admin_api.py

```diff
@@ -1,5 +1,6 @@
 import json
```

Comment:
The formatting looks a bit off here. Python linters & formatters should be catching this via pre-commit hooks

Reply:
This file did get pulled up for manual fixes, so whatever ended up in here went through the linters.  If there's anything in particular you're thinking of, I can sort it manually or look into which rules to enable.

Reply:
There seem to be spaces in the first part of `user_adds_user_invite_via_api `, `user_adds_release_role_to_user_via_api ` & `user_adds_publication_role_to_user_via_api `. Aside from that, all looks good

Reply:
Going to sort these formatting issues in a separate PR as it affects quite a few python files